evaluations.css

```diff
@@ -0,0 +1,97 @@
+		body {
+			min-width: 100%;
+			position: relative;
+			font-family: Arial;
+			background-color: #ffffff; 
+			margin: auto;
+		}
+
+		header {
+			height: 70px;
+			border-bottom: 1px solid grey;
+		}
+
+		footer {
+			height: 60px; 
+			position: absolute;
+			left: 0;
+			right: 0;
+			line-height: 60px;
+			background-color: #bcbcaf;
+			color: #686865;
+			font-family: Arial;
```

Comment:
Yep, it does, but apparently I forgot to delete that line after I set the font-family property for the body. Thanks!